src/app/detail/ui/styledComponent.tsx

```diff
@@ -91,12 +91,6 @@ export const TapWriteComment = styled(Tapwrite)(({ theme }) => ({
 }))
 
 export const TapWriteTaskEditor = styled(Tapwrite)(({ theme }) => ({
-  minHeight: '30vh',
+  minHeight: '10vh',
   fontSize: '20px',
 }))
```

Comment:
Let's use the new prop `editorClass` to adjust the height of the Tiptap's `Editor`.

Reply:
I think we need to refactor tiptap editor props to work consistently within our application